### ISSUE TITLE
Field Blocked by Message

### ISSUE THREAD

**Issue by Nzzyme** [AdguardTeam/AdguardForAndroid#3779]
Please see the screenshot. When ad blocking is disabled for an app, we are unable to see the last field (Size) in the filtering log for that entry.

Please fix. Thanks

![Screenshot_20210311235555](https://user-images.githubusercontent.com/6959116/110836069-a26aab80-82c5-11eb-8dff-678c13135025.jpg)

**Comment by anastenko** [comment]
Hello.
We will soon release version 4.0 and will take it into consideration.
Thanks for your suggestion.

**Comment by Nzzyme** [comment]
Thanks. You may want to tag it with a Milestone 4.0 then?

**Comment by zzebrum** [comment]
I wouldn't say it will  happen _soon_ 🙈 
Anyway, after total redesign the new task will be created with new UI bugs and improvements.

**Comment by Nzzyme** [comment]
So the app will go through a complete redesign with regards to the GUI?

**Comment by Chinaski1** [comment]
> So the app will go through a complete redesign with regards to the GUI?

Yes, in version 4.0. the interface will be completely redesigned.
